nifi-extension-bundles/nifi-snowflake-bundle/nifi-snowflake-services/src/main/java/org/apache/nifi/snowflake/service/SnowflakeComputingConnectionPool.java

```diff
@@ -252,6 +278,10 @@ protected Map<String, String> getConnectionProperties(final ConfigurationContext
         if (warehouse != null) {
             connectionProperties.put("warehouse", warehouse);
         }
+        if (tokenProvider != null) {
+            connectionProperties.put("authenticator", "oauth");
+            connectionProperties.put("token", tokenProvider.getAccessDetails().getAccessToken());
```

Comment:
It would be useful to define a static variable named `TOKEN_PROPERTY = "token"` for reuse here and in the refresh method.

Reply:
Actually, the `SFSessionProperty` enumeration can be used instead of these string property names.